.kitchen.yml

```diff
@@ -19,3 +19,4 @@ suites:
       - recipe[sudo]
       - recipe[user::data_bag]
       - recipe[osl-app::app1]
+      - recipe[osl-app::app2]
```

Comment:
Oh, I missed that there were multiple suites. I see that the user attribute for the app1 suite is 'iam' - does this username matter? 'iam' is not a username we're actually using on app1 or for iam (we have iam-staging and iam-production)